.semaphore/semaphore-scheduled-builds.yml

```diff
@@ -253,10 +255,12 @@ blocks:
       matrix:
       - env_var: ARCH
         values: [ "arm64", "armv7", "ppc64le", "s390x" ]
+      - env_var: LDFLAGS
```

Comment:
Also looks like you're passing LDFLAGS but the Makefile changes expect RELEASE_LDFLAGS?